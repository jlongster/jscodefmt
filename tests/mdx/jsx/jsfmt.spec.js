run_spec(__dirname, ["mdx"]);
