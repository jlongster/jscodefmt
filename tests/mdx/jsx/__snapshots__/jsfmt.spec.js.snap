// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`issue-9503.mdx format 1`] = `
====================================options=====================================
parsers: ["mdx"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
<ExternalLink href="http://aws.amazon.com/codepipeline/">AWS CodePipeline</ExternalLink> is a continuous delivery and release automation service that aids smooth deployments. You can integrate 3rd party tools into any step of your release process or you can use CodePipeline as an end-to-end solution, including Ghost Inspector tests and suites.

=====================================output=====================================
<ExternalLink href="http://aws.amazon.com/codepipeline/">
  AWS CodePipeline
</ExternalLink> is a continuous delivery and release automation service that
aids smooth deployments. You can integrate 3rd party tools into any step of your
release process or you can use CodePipeline as an end-to-end solution, including
Ghost Inspector tests and suites.

================================================================================
`;

exports[`issue-9503-2.mdx format 1`] = `
====================================options=====================================
parsers: ["mdx"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
<em>hey</em>

<em>hey</em>is a continuous

<em>hey</em> is a continuous


=====================================output=====================================
<em>hey</em>

<em>hey</em>is a continuous

<em>hey</em> is a continuous

================================================================================
`;
